pydicom/dicomdir.py

```diff
@@ -123,5 +123,14 @@ def get_siblings(record, map_offset_to_record):
                     child = map_offset_to_record[child_offset]
                     record.children = get_siblings(child, map_offset_to_record)
 
+        # Find the first patient record
+        first_patient_record = records[0]
+        for record in records:
+            if hasattr(record, 'DirectoryRecordType'):
```

Comment:
```python
first = [rr for rr in records if getattr(rr, 'DirectoryRecordType') == 'PATIENT']
if not first:
    # raise
```

Reply:
Right - shouldn't have just copied that...

Reply:
Ok, done.